setup.py

```diff
@@ -13,5 +13,26 @@
     author_email='developers@savannahinformatics.com',
     url='',
     packages=find_packages(),
-    install_requires=[],
+    install_requires=[
+        "django==1.8.0",
+        "Fabric==1.10.0",
+        "coverage==3.7.1",
+        "psycopg2==2.5.4",
+        "djangorestframework==3.1.0",
+        "django-filter==0.9.2",
+        "markdown==2.5.1",
+        "python-dateutil==2.4.0",
+        "flake8==2.3.0",
+        "django-grappelli==2.6.3",
```

Comment:
Remove `grapelli` - both here and in settings ( if applicable ).

We are completely getting rid of the admin "crutch".